sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -312,3 +315,90 @@ case class Logarithm(left: Expression, right: Expression)
     """
   }
 }
+
+case class Round(child: Expression, scale: Expression) extends Expression {
+
+  def this(child: Expression) = {
+    this(child, Literal(0))
+  }
+
+  def children: Seq[Expression] = Seq(child, scale)
+
+  def nullable: Boolean = true
```

Comment:
I mean `nullable` is sometimes determined at runtime, a not null string, double.NaN is not null themselves, but would `eval` to `null` in `Round`.